bittensor/metagraph.py

```diff
@@ -799,11 +780,10 @@ def save(self) -> "metagraph":
         """
         save_directory = get_save_dir(self.network, self.netuid)
         os.makedirs(save_directory, exist_ok=True)
-        graph_file = save_directory + f"/block-{self.block.item()}.pt"
+        graph_filename = save_directory + f"/block-{self.block.item()}.pt"
```

Comment:
```suggestion
        graph_filename = save_directory + f"/block-{self.block.item()}.pickle"
```
pt is pytorch extension,

Reply:
since pt files are also pickle files and we want load to support torch saves, it's simpler to leave it as pt

Reply:
we want support importing from legacy files, that doesn't mean we want to keep pretending we still use the same format, but do as you will